#!/usr/bin/python3
# 6-raise_exception_msg.py

def raise_exception_msg(message=""):
    """Function that raises a NameError with a message."""
    raise NameError(message)
